staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -98,8 +98,9 @@ func (d *validatingDispatcher) Dispatch(ctx context.Context, attr *generic.Versi
 
 func (d *validatingDispatcher) callHook(ctx context.Context, h *v1beta1.Webhook, attr *generic.VersionedAttributes) error {
 	if attr.IsDryRun() {
-		// TODO: support this
-		return webhookerrors.NewDryRunUnsupportedErr(h.Name)
+		if h.SideEffects == nil || *h.SideEffects == v1beta1.SideEffectClassUnknown || *h.SideEffects == v1beta1.SideEffectClassSome {
```

Comment:
same question here... requiring a positive match on an enum value we understand that means dry run is safe seems better